cs4090/courseApp/models/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from django.contrib.auth.models import (
+    AbstractBaseUser,
+    BaseUserManager,
+    PermissionsMixin,
+)
+from django.utils.translation import gettext_lazy as _
+from django.utils import timezone
+from django.conf import settings
+
+
+class Course(models.Model):
+    # ForeignKey to the User model, making each class unique to a user
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name="courses"
+    )
+
+    # Fields for class details
+    name = models.CharField(max_length=100, help_text="Course name, e.g., Math 101")
+    start_time = models.TimeField(help_text="Course start time")
+    end_time = models.TimeField(help_text="Course end time")
+
+    DAYS_OF_WEEK_CHOICES = [
+        ("Mon", "Monday"),
+        ("Tue", "Tuesday"),
+        ("Wed", "Wednesday"),
+        ("Thu", "Thursday"),
+        ("Fri", "Friday"),
+        ("Sat", "Saturday"),
+        ("Sun", "Sunday"),
+    ]
+    # Comma-separated list of days (or use another approach for multiple days)
+    days_of_week = models.CharField(
```

Comment:
This is the only thing I don't like.

It doesn't make sense to type in abbreviated names. Either selecting boxes or letters would be better imo. (Something like checking Monday, Wednesday, Friday)
![checl](https://github.com/user-attachments/assets/00d6787c-49a5-42d1-a6d0-24945d0095c2)

Reply:
It is done